src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php

```diff
@@ -1878,7 +1878,7 @@ public function testIfNotifierTransportsAreKnownByFrameworkExtension()
     {
         $container = $this->createContainerFromFile('notifier');
 
-        foreach ((new Finder())->in(\dirname(__DIR__, 4).'/Component/Notifier/Bridge')->directories()->depth(0)->exclude('Mercure') as $bridgeDirectory) {
+        foreach ((new Finder())->in(\dirname(__DIR__, 4).'/Component/Notifier/Bridge')->directories()->depth(0)->exclude(['Mercure']) as $bridgeDirectory) {
```

Comment:
Why this change?

Reply:
Ah its based on https://github.com/symfony/symfony/pull/41713#discussion_r653378795

Can you please remove the `[]`? Thanks